src/Builder.php

```diff
@@ -372,6 +372,25 @@ public function withClaim($name, $value)
 
         return $this;
     }
+    
+    /**
+     * Configures some claim items
+     *
+     * @param array $claims
+     *
+     * @return Builder
+     */
+    public function withClaims($claims)
+    {
+        if(!is_array($claims) || empty($claims)){
+            return $this;
```

Comment:
This is not a fail-safe way to handle invalid input. It will cause silent error, user has no any notification about invalid input. Please use method type hint for argument `$claims` like this:
```php
public function withClaims(array $claims)
```